### PR TITLE
Fix typos in cargo-specifics.md

### PR DESCRIPTION
**Pull request by cakebaker**
This PR fixes three typos in `cargo-specifics.md`.

### CODE REVIEW

**Comment by lqd** [comment]
Could you fix https://github.com/rust-lang/rust/pull/125494/files#diff-7a45221cce23aa10d6ed6e4820bf30f70bc8ba96415b8e4381a7089f05831acbR6 as well?

File: src/doc/rustc/src/check-cfg/cargo-specifics.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ implementation detail and is therefor not documented in Cargo, we therefor do th
 
 *See the [`[lints]` section in the Cargo book][cargo-lints-table] for more details.*
 
-When using a staticlly known custom config (ie. not dependant on a build-script), Cargo provides
+When using a statically known custom config (ie. not dependant on a build-script), Cargo provides
```
Comment (lqd) [inline src/doc/rustc/src/check-cfg/cargo-specifics.md:40]: here, it's probably even

```suggestion
When using a statically known custom config (ie. not dependent on a build-script), Cargo provides
```

**Comment by ehuss** [comment]
r? ehuss

**Comment by ehuss** [comment]
Closing in favor of https://github.com/rust-lang/rust/pull/126048 which contains the additional edits in order to get this before beta. Thanks @cakebaker!

**Pull request closed without merging**